public/scripts/views/forms/upload.js

```diff
@@ -108,7 +108,7 @@
           expression.parse(element.dataset["write"] || "[]")
         );
 
-        sdk.storage.createFile('unique()', files[0], read, write, 1).then(
+        sdk.storage.createFile('default', 'unique()', files[0], read, write, 1).then(
           function(response) {
```

Comment:
Any reason we chose to have the bucket as the first param in create file  ? 🤔 
Every other  Appwrite endpoint that creates something, expects the ID first 🤔 .
Even the `createDocument` endpoint. What do you think ?

Reply:
I was not aware of this change.

Reply:
I don't think that's the case,  as collectionId, bucketId being the path param it comes before body param
https://appwrite.io/docs/client/database?sdk=web-default#databaseCreateDocument